### ISSUE TITLE
SOSO guidance should provide an identifier for metadata that conforms to the spec

### ISSUE THREAD

**Issue by smrgeoinfo** [ESIPFed/science-on-schema.org#169]
It would be useful for client applications to know that a given metadata record conforms to the science on schema.org recommendations.   Currently 'conformsTo' is a [schema.org property under discussion](https://github.com/schemaorg/schemaorg/issues/1516), but dct:conformsTo is already widely used. 
Usage would be like:
```
 {
  "@context": {
    "@vocab": "https://schema.org/"
  },
  "@id": "metadata:lod.example-data-repository.org/id/dataset/3300",
  "dcterms:conformsTo":"https://github.com/ESIPFed/science-on-schema.org/1.2.0",
  "identifier":"https://doi.org/10.1234/1234567890",
```

In the DataSet.md guidance document, we'd need to add some introductory text stating that 
"To identify a metadata record as conforming to the recommendations in this document, use the URI 'https://github.com/ESIPFed/science-on-schema.org/1.2.0'  as in the example above.

**Comment by charlesvardeman** [comment]
There is also the [Profiles Vocabulary](https://www.w3.org/TR/dx-prof/) from the W3C Dataset Exchange Working Group. The Profiles Vocabulary uses dcterms:conformsTo connecting an rdf:Resource to a dcterms:Standard. Profile is a subclass of dcterms:Standard so that would work for science-on-schema.org except I think it needs to become a URI. 

```jsonld
 { 
"@context": {
    "@vocab": "https://schema.org/"
  },
  "@id": "metadata:lod.example-data-repository.org/id/dataset/3300",
  "dcterms:conformsTo":[{"@id":"https://github.com/ESIPFed/science-on-schema.org/1.2.0"}],
  "identifier":"https://doi.org/10.1234/1234567890"
}
```

That would allow science-on-schema.org the option to create a dereferenceable  identifier in the future where <https://github.com/ESIPFed/science-on-schema.org>  prof:isProfileOf <https://schema.org/> .

I think science-on-schema.org meets the definition of a prof:Profile, I.e.,  "A specification that constrains, extends, combines, or provides guidance or explanation about the usage of other specifications."

**Comment by smrgeoinfo** [comment]
@charlesvardeman we're on the same page. What we need is that URI for the SOSO schema.org profile. It needs to be explictly stated in the guidance document.  I think 'https://github.com/ESIPFed/science-on-schema.org/1.2.0' would be a bad choice for the profile URI. It should at least be a github tag.   See also #170  for question as to whether the value should be a LOD URI, or just a URI string. What are your thoughts about that?

**Comment by smrgeoinfo** [comment]
I see that [DataOne is using something like this](https://cn.dataone.org/cn/v2/query/solr/?q=formatId:science-on-schema*+AND+formatType:METADATA+AND+-obsoletedBy:*&fl=identifier,id,size,formatId,checksum,checksumAlgorithm,replicaMN,dataUrl,datasource&rows=100&wt=json) to identify metadata conforming to SOSO 'profile':
`"formatId": "science-on-schema.org/Dataset;ld+json"`

**Comment by mbjones** [comment]
Yes, that comes from our format vocabulary that we use for typing METADATA and DATA format types consistently across the federation:

https://github.com/DataONEorg/object-formats/
